org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/desktop/outline/pages/IPageWithNodes.java

```diff
@@ -25,4 +25,9 @@
 public interface IPageWithNodes extends IPage<ITable> {
 
   void rebuildTableInternal();
+
+  /**
+   * Updates the table row with the content of the page .
+   */
+  void updateTableRowFromPage(IPage page);
```

Comment:
Please update JavaDoc of updateCellFromPageCell. Remove "Called when the table gets rebuilt" because this is not always true anymore. Also remove the _row._ at the end